backend/api/src/create-market.ts

```diff
@@ -615,7 +615,7 @@ async function generateAntes(
     drizzledAmount > 0 &&
     (contract.mechanism === 'cpmm-1' || contract.mechanism === 'cpmm-multi-1')
   ) {
-    return await pg.tx(async (tx) => {
+    return await pg.txIf(async (tx) => {
```

Comment:
Ooh, don't know what this does

Reply:
it creates a new transaction iff it is not called from within another transaction